alluxio-ui/worker/config-overrides.js

```diff
@@ -30,9 +30,11 @@ module.exports = {
     // ...add your webpack config customisation, rewires, etc...
     const srcPath = resolveApp('src');
     const commonUiSrcPath = resolveApp('../common/src');
-    config.module.rules[0].include = [srcPath, commonUiSrcPath];
-    config.module.rules[1].oneOf[1].include = [srcPath, commonUiSrcPath];
-    config.module.rules[1].oneOf[2].include = [srcPath, commonUiSrcPath];
+    config.resolve.plugins[1].appSrcs = [srcPath, commonUiSrcPath];
+    config.module.rules[1].include = [srcPath, commonUiSrcPath];
+    config.module.rules[2].oneOf[1].include = [srcPath, commonUiSrcPath];
+    config.plugins[9].options.watch = [srcPath, commonUiSrcPath];
+    config.plugins[9].watch = [srcPath, commonUiSrcPath];
```

Comment:
Why is this the 9th element?

Reply:
This is a hack, the build happens under the hood (create-react-app internals), but we want to be able to have some source files above the src directory. There are two ways to do this:

1. Eject the application and handle the build ourselves (modifying the webpack configuration).
2. Modify the configuration via a rewire as the application resolves it dependencies.

We do the second here, and the JSON configuration affecting your example will contain an array for plugins in which we modify the 9th element.

In the future, this will break in the future when we update the react-scripts dependency, but we should be able to print out what the new configuration looks like and update this there (unless the react team accepts monorepos at that point, which will deprecate this).